@here/harp-mapview/lib/MapView.ts

```diff
@@ -1818,25 +1870,39 @@ export class MapView extends THREE.EventDispatcher {
         this.m_camera.updateProjectionMatrix();
         this.m_camera.updateMatrixWorld(false);
 
+        this.m_rtcCamera.copy(this.m_camera);
+        this.m_rtcCamera.position.sub(this.m_camera.position);
+        this.m_rtcCamera.updateProjectionMatrix();
```

Comment:
looking at https://github.com/mrdoob/three.js/blob/dev/src/cameras/Camera.js, it seems that the projectionMatrix is copied over from the this.m_camera, so it should not be theoretically necessary? maybe I miss something.